package.json

```diff
@@ -1,30 +1,34 @@
 {
-  "name": "telraam-bloemekens-sveltekit",
```

Comment:
Most changes are related to running `npm run format`. To check the changes without whitespaces, use https://github.com/stijnvanhoey/telraam-bloemekeswijk/pull/6/files?w=1.